tests/e2e/models/CMakeLists.txt

```diff
@@ -85,3 +85,79 @@ iree_check_single_backend_test_suite(
 )
 
 ### BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE ###
+
+iree_static_linker_test(
+  NAME
+    mnist_fake_weights_llvm_cpu_static_bytecode_test
+  SRC
+    "mnist_fake_weights.mlir"
+  STATIC_LIB_PREFIX
+    mnist_fake_weights_linked_llvm_cpu
+  MAIN_FUNCTION
+    "module.predict"
+  INPUT_SHAPES
+    "1, 28, 28, 1"
+  INPUT_TYPE
+    float
```

Comment:
Can we use the same `"1x320x320x3xf32"` format as in other parts of the project here? https://github.com/iree-org/iree/blob/e33c64c8c1b58958cf56982372e53332c5c1aeb2/benchmarks/TFLite/CMakeLists.txt#L41-L45

Reply:
Done.